python/whylogs/core/schema.py

```diff
@@ -164,6 +173,13 @@ class ColumnSchema:
     cfg: MetricConfig = MetricConfig()
     type_mapper: TypeMapper = StandardTypeMapper()
     resolver: Resolver = StandardResolver()
+    validators: Dict[str, Validator] = field(default_factory=dict)
 
     def get_metrics(self, name: str) -> Dict[str, Metric]:
         return self.resolver.resolve(name=name, why_type=self.type_mapper(self.dtype), column_schema=self)
+
+    def get_validators(self, name: str) -> Set[Validator]:
+        try:
+            return self.validators[name]
+        except KeyError:
+            return set()
```

Comment:
Avoid using exception handling for missing keys, better to use something like:
```suggestion
        if self.validators:
            return self.validators.get(name)
```

Reply:
done in the latest commits